### PR TITLE
refactor!: use whiskers, add accents & support latest versions

### PR DESCRIPTION
**Pull request by Covkie**
changes ripped out (not pretty but works) from https://github.com/Covkie/ctp-discord/tree/36ee62d6fcd67f1ecc6a2be579ea27dd000f513a/src/react-native. 

as discussed in https://discord.com/channels/907385605422448742/1221083282146332742

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,81 @@
 <h3 align="center">
 	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/logos/exports/1544x1544_circle.png" width="100" alt="Logo"/><br/>
 	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
-	Catppuccin for <a href="https://github.com/vendetta-mod/Vendetta">Vendetta</a>
+	Catppuccin for <a href="https://github.com/pyoncord/Bunny">Bunny</a> & <a href="https://github.com/enmity-mod/enmity">Enmity</a>
```
Comment (sgoudham) [inline README.md:4]: I think we should still mention Vendetta in this list.

File: README.md [hunk 2]
```diff
@@ -1,76 +1,81 @@
...
-<img src="https://raw.githubusercontent.com/riivx/vendetta//main/assets/mocha.webp"/>
+<summary>🌿 Mocha</summary> 
+<img src="assets/mocha.webp"/>
 </details>
 
 ## Usage
 
->[!WARNING]
->Vendetta was **discontinued** on February 7. 2024. Use [Bunny](https://github.com/pyoncord/Bunny) instead. It supports Vendetta themes
-
```
Comment (sgoudham) [inline README.md:41]: I think we should keep this text but reword it to explain the point of this repository, and under `[!IMPORTANT]` instead.

File: README.md [hunk 3]
```diff
@@ -1,76 +1,81 @@
...
+>
+> If you are using Enmity, install [VendettaCommpat](https://discord.com/channels/950850315601711176/1019645460589318165/1211473409154744350) plugin first. After doing so, follow this [guide](https://discord.com/channels/950850315601711176/1239393517151846412) as it loads Vendetta by default which is broken past Discord version 223
+
+## Fonts
+### Bunny
+1. Head to `Settings` > `Bunny` > `Fonts` > `add button` > Extract font from theme
+2. Click `Extract` > `Import` Then select `Meloso`
+
+### Enmity
+ Head over to [Rosie's repo](https://github.com/acquitelol/rosiecord?tab=readme-ov-file#how-to-install) and follow the steps provided.
```
Comment (sgoudham) [inline README.md:70]:
```suggestion
 Head over to [acquitelol/rosecord](https://github.com/acquitelol/rosiecord?tab=readme-ov-file#how-to-install) and follow the steps provided. 
```

**Review by sgoudham — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sgoudham) [reply README.md:4]: The name of the repository will still be `vendetta` so I'd personally prefer if you could include it within this list. We can mention that Vendetta is EOL later on in the README

File: README.md [hunk 4]
```diff
@@ -1,76 +1,84 @@
...
+>
+> If you are using Enmity, install [VendettaCommpat](https://discord.com/channels/950850315601711176/1019645460589318165/1211473409154744350) plugin first. After doing so, follow this [guide](https://discord.com/channels/950850315601711176/1239393517151846412) as it loads Vendetta by default which is broken past Discord version 223
 
+## Fonts
+### Bunny
+1. Head to `Settings` > `Bunny` > `Fonts` > `add button`
+2. Follow shown options
 > [!NOTE]
-> If the icon pack doesn't apply after discord loads, then switch to a different server and back.
+> The font pack shown in the screenshots is [Flow Circular](https://gist.githubusercontent.com/Covkie/d4124cfc4df15e7790167ac88e820e0d/raw/02f0c7124f233ee236f0682d464c7cb22c2ea6e9/FlowCircular-font-snippet.json)
```
Comment (riichi67) [inline README.md:72]: preview of the theme has no text at all, so we should probably move this somewhere

Comment (riichi67) [reply README.md:72]: my bad, then we're keeping it (don't know who will use that, but still)

**Comment by riichi67** [comment]
just pushed two new fixes, but those are related to fonts and backgrounds. also wanted to ask if we're going to do something with custom fonts or just stay vanilla

**Comment by riichi67** [comment]
if we're removing fonts support, that solves #29

**Comment by sgoudham** [comment]
I've updated the README to mention Vendetta as I think it's important since the repository name is `catppuccin/vendetta`. I'd quite like to merge this and get the ball rolling so I'd appreciate if someone else could resolve the merge conflicts :+1:

**Comment by riichi67** [comment]
i'll work on resolving conflicts and remove fonts as well

**Review by riichi67 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,87 +1,94 @@
...
 1. Choose a flavor, and copy its link below:
-	- **Latte**: https://raw.githubusercontent.com/catppuccin/vendetta/main/latte.json
-	- **Frappe**: https://raw.githubusercontent.com/catppuccin/vendetta/main/frappe.json
-	- **Macchiato**: https://raw.githubusercontent.com/catppuccin/vendetta/main/macchiato.json
-	- **Mocha**:  https://raw.githubusercontent.com/catppuccin/vendetta/main/mocha.json
-2. Head to `You Tab (aka Settings)` > `Vendetta` > `Themes`
+   - **Latte**: <https://raw.githubusercontent.com/catppuccin/vendetta/main/latte.json>
+   - **Frappe**: <https://raw.githubusercontent.com/catppuccin/vendetta/main/frappe.json>
+   - **Macchiato**: <https://raw.githubusercontent.com/catppuccin/vendetta/main/macchiato.json>
+   - **Mocha**: <https://raw.githubusercontent.com/catppuccin/vendetta/main/mocha.json>
```
Comment (sgoudham) [inline README.md:51]: I think this step should be changed to the following:

```md
1. Choose a flavor + accent combination from the [themes](./themes) directory and copy the URL in the following format: `https://raw.githubusercontent.com/catppuccin/vendetta/main/themes/<flavor>/<accent>.json`
	
	For example, if you'd like to use the `mocha` flavor with the `mauve` accent, copy the following link:

	```
	<https://raw.githubusercontent.com/catppuccin/vendetta/main/themes/mocha/mauve.json>
	```
```

Doesn't have to be exactly that ^ but something similar and equivalent to where we showcase the format and give one example.

Comment (riichi67) [reply README.md:51]: > Doesn't have to be exactly that ^ but something similar and equivalent to where we showcase the format and give one example.

imo it should be exactly that

**Review by sgoudham — APPROVED** [review]
Looks good to me from an org perspective, thanks to everyone who contributed to this PR.
